### DISCUSSION TITLE
How to install stable-latest version of Python automatically?

### DISCUSSION THREAD

**Discussion by solzard** [pyenv/pyenv#2254] [Q&A]
`pyenv install 3:latest` even selects a dev build (e.g. 3.11.0a4 as of Feb 2022). How can I let `pyenv` to find a stable latest build (e.g. 3.10.2 as of Feb 2022)?

**Comment by ssbarnea** [comment]
While I seen the feature being added, it does not seem to work at all, even with pyenv 2.3.9

**Comment by anton-petrov** [comment]
https://github.com/pyenv/pyenv#prefix-auto-resolution
